docs/ref/models/fields.txt

```diff
@@ -502,6 +502,15 @@ Any combination of these options will result in an error.
     ``True`` will cause the field to have ``editable=False`` and ``blank=True``
     set.
 
+.. note::
+    When using timezones, storing a ``DateField`` in the database is
+    ambiguous because it's incapable of storing timezone aware information, unlike ``DateTimeField``.
+    Unless you know you will need ``DateField``, consider using ``DateTimeField`` instead
+    and convert to a date format for display only.
+
+    ``auto_now`` and ``auto_now_add`` become ambiguous as well.
+    It's recommended to set a ``default`` value that does what you expect regarding timezones.
```

Comment:
Overriding `save()` provides a better implementation of what users of `auto_now(_add)` expect.